packages/adblocker/src/filters/network.ts

```diff
@@ -451,6 +451,83 @@ function getFilterOptionValue(line: string, pos: number, end: number): [number,
   return [pos, value];
 }
 
+/**
+ * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Character_escape
+ * \f, \n, \r, \t, \v, \0, \^, \$, \\, \., \*, \+, \?, \(, \), \[, \], \{, \}, \|, \/
+ * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Character_class_escape
+ * \d, \D, \s, \S, \w, \W
+ * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Word_boundary_assertion
+ * \b, \B
+ */
+const REGEXP_CHARACTER_ESCAPES = new Set([
+  102, 110, 114, 116, 118, 48, 94, 36, 92, 46, 42, 43, 63, 40, 41, 91, 93, 123, 125, 124, 47, 100,
+  68, 119, 87, 115, 83, 98, 66,
```

Comment:
can we change in this way? 

```
  98, // b
  102, // f
  110, // n
  114, // r
  116, // t
  118, // v
  94, // ^
  36, // $
  134, // \
  46, // .
  42, // *
  43, // +
  63, // ?
  40, // (
  41, // )
  91, // [
  93, // ]
  123, // {
  125, // }
  124, // |
  92, // /
  83, // S
  115, // s
  119, // w
```

Reply:
Fixed in [`9b0d3ea` (#4241)](https://github.com/ghostery/adblocker/pull/4241/commits/9b0d3ea09f1671f1a0c5cf749d731fc5e92d67ad)